c/enc/block_splitter_inc.h

```diff
@@ -129,7 +129,7 @@ static size_t FN(FindBlocks)(const DataType* data, const size_t length,
     }
     /* More blocks for the beginning. */
     if (byte_ix < 2000) {
-      block_switch_cost *= 0.77 + 0.07 * (double)byte_ix / 2000;
+      block_switch_cost *= 0.77 + 0.00003 * (double)byte_ix;
```

Comment:
Sorry, missed a digit.
```suggestion
      block_switch_cost *= 0.77 + 0.000035 * (double)byte_ix;
```